### PR TITLE
rationalizing logs

### PR DESCRIPTION
**Pull request by heckj**
updating mapping to more constricted set of logs per
https://github.com/RackHD/RackHD/issues/34

**NOTE**: Requires https://github.com/RackHD/on-core/pull/48 to be
committed prior as it adds the synonym 'critical' to the existing 'crit'
level.

### CODE REVIEW

File: lib/services/ws-service.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ function webSocketServiceFactory(
     };
 
     WebSocketService.prototype._forwardResourceMethod = function (method, msg, wsConn, rawMsg) {
-        logger.info('Client: %s requested a resource call.'.format(wsConn.id),
+        logger.debug('Client: %s requested a resource call.'.format(wsConn.id),
```
Comment (heckj) [inline lib/services/ws-service.js]: @rolandpoulter this sequence seemed more relevant to `debug` than `info`, so I'm suggesting we switch them over. That work for you?

**Comment by DavidjohnBlodgett** [comment]
:+1:

File: lib/services/ws-service.js (hunk 1 above)
Comment (rolandpoulter) [inline lib/services/ws-service.js]: Yup, totally agree.

**Comment by rolandpoulter** [comment]
:+1:

**Comment by yyscamper** [comment]
:+1:

**Pull request merged**